src/components/CovMap.tsx

```diff
@@ -75,6 +76,7 @@ export const CovMap = () => {
   const dispatch = useThunkDispatch();
   const urlParams = useParams<{ subPage?: string }>();
   const history = useHistory();
+  const isMobile = useMediaQuery("(max-width:600px)");
```

Comment:
just one remark can we centralize the break-points somewhere?

Reply:
Yes, was thinking about that as well... would like to keep it as a reminder for #281